web/client/components/app/__tests__/StandardApp-test.jsx

```diff
@@ -290,9 +293,10 @@ describe('StandardApp', () => {
         const MyApp = ({plugins}) => {
             expect(plugins.MyPlugin).toExist();
             done();
+            return null;
         };
 
-        const app = ReactDOM.render(<StandardApp appComponent={MyApp} appStore={store} enableExtensions />, document.getElementById("container"));
+        const app = ReactDOM.render(<StandardAppWithExtensions appComponent={MyApp} appStore={store} enableExtensions />, document.getElementById("container"));
```

Comment:
I don't know if all these tests require the extensions hoc to be applied. If not, maybe we should use it only for tests related ot extensions behaviour. What do you think?

Reply:
@mbarto ok, I'll double check them. Should I move them in a new withExtensions-test file?

Reply:
yes, maybe it would be better

Reply:
moved test to withExtensions-test